src/hermes.jsx

```diff
@@ -348,6 +351,9 @@ function getSelectedNode () {
 }
 
 function selectionChanged (prevSel, sel) {
+	if (prevSel === null && sel === null) {
```

Comment:
Oh yeah, my bad.  So what causes both of these to be `null`?  Because it seems like that should be fixed outside of this function.